docs/source/optimization/params.rst

```diff
@@ -0,0 +1,91 @@
+.. _params:
+
+====================
+The params DataFrame
+====================
+
+The ``params`` DataFrame is an important concept in estimagic. It collects
+information on the dimensionality of the optimization problem, lower and upper
+bounds, categories of parameters and valid ranges for randomly sampled parameter
+vectors. Moreover, it is the mandatory first argument of any criterion function
```

Comment:
the criterion function only use Series ... at least that's what's written in the documentation of minimize and maximize.

Reply:
This was changed months ago!!! I found one outdated docstring, but hope that's it.